knex/migrations/20190219194834_users.js

```diff
@@ -3,6 +3,7 @@ exports.up = function (knex, Promise) {
   return knex.schema.createTable('users', function (table) {
     table.increments().primary().unique().notNullable();
     table.string('username', 255).unique().notNullable();
+    table.string('password').notNullable();
```

Comment:
Instead of editing this file, you should make a new migration for adding the password to the uses database table.

Reply:
I tried making a new migration last night and it didn't work because of my seeds were interfering so I also tried deleting data from my table and then I tried it again but it wouldn't work for some reason.  Isaiah gave me the okay to just rollback.

Reply:
So you're database tables aren't going to have a password column if you do this right? Were you able to completely rollback and migrate again?

Reply:
Yes